lib/views/bidding_slack_view.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Aprb.Views.BiddingSlackView do
+  import ViewHelper
+
+  def render(event) do
+    artwork_data = fetch_sale_artwork(event["lotId"])
+    %{
+      text: ":gavel: #{event["type"]} on #{artwork_data[:permalink]}",
+      attachments: "[{
+                      \"fields\": [
+                        {
+                          \"title\": \"Amount\",
+                          \"value\": \"#{format_price((event["amountCents"] || 0) / 100)}\",
```

Comment:
This `/100` operation will get repeated a lot. It might be better if `format_price` accepted its argument in cents and did the conversion.